api/v1beta1/machinedeployment_types.go

```diff
@@ -211,6 +216,30 @@ type MachineRollingUpdateDeployment struct {
 
 // ANCHOR_END: MachineRollingUpdateDeployment
 
+// ANCHOR: RemediationStrategy
+
+// RemediationStrategy allows to define how the MachineSet can control scaling operations.
+type RemediationStrategy struct {
+	// MaxInFlight determines how many in flight remediations should the MachineSet issue at
+	// any given point in time.
+	//
+	// This fields gives control over how many Machines should be remediated (deleted),
+	// at any given point in time during the MachineSet's reconciliation.
+	//
+	// Remediation only happens on the MachineSet with the current revision, during rollout
+	// operations, older MachineSets aren't allowed to remediate while scaling is in progress.
```

Comment:
nit
```suggestion
	// operations, older MachineSets aren't allowed to remediate while scaling down is in progress
	// (however, unhealthy machines will be deleted with higher priority than healthy ones).
```

Reply:
Added the notes in parenthesis, I didn't add the `down` because it was actually during a rollout, so removed the `scaling` word